pkg/framework/host.go

```diff
@@ -99,18 +99,16 @@ func (h *Host) AWSCluster(name string) (*v1alpha1.AWSConfig, error) {
 	return cluster, nil
 }
 
-func (h *Host) DeleteGuestCluster() error {
-	if err := runCmd("kubectl delete awsconfig ${CLUSTER_NAME}"); err != nil {
+func (h *Host) DeleteGuestCluster(cr, name, logEntry string) error {
```

Comment:
It would be nice to align with the current Install methods and have the name parameter before the cr, see https://github.com/giantswarm/e2e-harness/blob/master/pkg/framework/host.go#L117